src/utils/types.ts

```diff
@@ -25,3 +26,29 @@ export type CloudInfoResult = {
   isGcp: boolean;
   isAzure: boolean;
 };
+
+export type NodeOptions = {
```

Comment:
I think we already have a place where it might make sense for this to live: https://github.com/mongodb-js/vscode/blob/master/src/connectionModelType.ts#L5

Kind of an aside, how do you feel about having a folder for these types, so instead of `utils/types.ts` it's `types/CollectionItem.ts` or`types/ConnectionModel.ts`.

For shell completion items is it possible to use them straight from mongosh instead of redefining them here?

With connection model maybe we can even go further and similar to how we treat `.less` files: https://github.com/mongodb-js/vscode/blob/master/src/views/webview-app/externals.ts but in this case we can define the `mongodb-connection-model`
we can create a type definition for that module and use that. Maybe that's a different part of work from this though we can do later. (Maybe we don't bother if we want to have our own connection model in this repo also).